Show how to get API keys (#166)
ArcadeAI/docs: pages/home/use-tools/call-tools-directly.mdx

### Prerequisites

- [Set up Arcade](/home/quickstart)
- Create an Arcade [API key](/home/api-keys), if you haven't already

### Set environment variables

You can find your Arcade API key in one of these locations:

> - [Arcade Dashboard](https://api.arcade.dev/dashboard/api-keys) under API Keys
> - `~/.arcade/credentials.yaml`
> 
> ```bash
> export ARCADE_API_KEY=<your-api-key>

Reviewer: ```suggestion
- [Arcade Dashboard](https://api.arcade.dev/dashboard/api-keys) under API Keys
- `~/.arcade/credentials.yaml`

Export your Arcade API key as an environment variable:
```bash
export ARCADE_API_KEY=<your-api-key>
```

### Set environment variables

You can find your Arcade API key in one of these locations:

- [Arcade Dashboard](https://api.arcade.dev/dashboard/api-keys) under API Keys
- `~/.arcade/credentials.yaml`

```bash
export ARCADE_API_KEY=<your-api-key>
```

> > Need an Arcade API key? Visit the [How to get an API key](/home/api-keys) page to create one.

Reviewer: Like the call-tools-with-models page, I think it makes sense to move the API key requirement to the Prerequisites section, since it's something you need to do before you follow the steps on the page.
